pilot/pkg/features/pilot.go

```diff
@@ -413,4 +413,8 @@ var (
 	StatusMaxWorkers = env.RegisterIntVar("PILOT_STATUS_MAX_WORKERS", 100, "The maximum number of workers"+
 		" Pilot will use to keep configuration status up to date.  Smaller numbers will result in higher status latency, "+
 		"but larger numbers may impact CPU in high scale environments.")
+
+	WasmRemoteLoadConversion = env.RegisterBoolVar("ISTIO_AGENT_ENABLE_WASM_REMOTE_LOAD_CONVERSION", true,
```

Comment:
Suggest set it false

Reply:
The logic won't be triggered if user does not use ECDS. The implementation of ECDS in istiod is only added at 1.9, there should be no ECDS usage before. So I think we are safe to just turn it on by default.